frontend/pages/hosts/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -144,8 +161,130 @@ export class HostDetailsPage extends Component {
     );
   };
 
+  // renderQueryHostModal = () => {
```

Comment:
This is the section I'm referring to in the above comment.

Reply:
100%